packages/dev/core/src/Shaders/ShadersInclude/openpbrEnvironmentLighting.fx

```diff
@@ -0,0 +1,174 @@
+// _____________________________ Base Diffuse Layer IBL _______________________________________
+#ifdef REFLECTION
+    // Pass in a vector to sample teh irradiance with (to handle reflection or )

```

Comment:
This comment is a bit hard to understand :)

Reply:
Indeed. That's a half-written comment.